src/index.html

```diff
@@ -1,13 +1,410 @@
+          </div>
+          <!--  2:   -->
+          <!--  IMG:   -->
+          <div class="testimonials__item grid__item grid__item--5-8">
+            <div class="testimonial">
+              <div class="testimonial__img-box">
+                <img
+                  src="./images/photos/p2.png"
+                  alt="p1"
+                  class="testimonial__img"
+                >
+              </div>
+              <!--  NAME:   -->
+              <div class="testimonial__content">
+                <a
+                  href="https://uk.linkedin.com/in/scarlett-hawkins-42b442132"
+                  class="testimonial__name"
+                >
+                  Scarlett Hawkins
+                </a>
+                <span class="testimonial__role">Senior Vice President</span>
+                <p class="testimonial__phrase">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+              </div>
+            </div>
+          </div>
+          <!--  3:   -->
+          <!--  IMG:   -->
+          <div class="testimonials__item grid__item grid__item--9-12">
+            <div class="testimonial">
+              <div class="testimonial__img-box">
+                <img
+                  src="./images/photos/p3.png"
+                  alt="p1"
+                  class="testimonial__img"
+                >
+              </div>
+              <!--  NAME:   -->
+              <div class="testimonial__content">
+                <a
+                  href="https://www.linkedin.com/in/maxghenry"
+                  class="testimonial__name"
+                >
+                  Max Henry
+                </a>
+                <span class="testimonial__role">Chief Design Officer</span>
+                <p class="testimonial__phrase">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <h2 id="contact" class="page__title">
+        Contact us
+      </h2>
+
+      <section class="page__section contacts">
+        <div class="grid grid--desktop">
+          <div class="grid__item grid__item--1-2">
+            <div class="contact">
+              <p class="contact__value-name">
+                Phone
+              </p>
+              <p class="contact__value">
+                +1 234 5555-55-55
+              </p>
+              <p class="contact__value-name">
+                Email
+              </p>
+              <p class="contact__value">
+                hello@miami.com
+              </p>
+            </div>
+          </div>
+          <div class="grid__item grid__item--4-5">
+            <div class="contact">
+              <p class="contact__value-name">
+                Address
+              </p>
+              <p class="contact__value">
+                400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401
+              </p>
```

Comment:
Checklist rule №4 Change text color on hover for phone, email and address. 
Also it's not just a text, it's links (tel:, mailto:, and google maps)